src/services/inlayHints.ts

```diff
@@ -267,6 +267,14 @@ namespace ts.InlayHints {
             if (!signature) {
                 return;
             }
+            const wrapParen = isArrowFunction(node) && !findChildOfKind(node, SyntaxKind.OpenParenToken, file);
+            if (wrapParen) {
+                result.push({
+                    text: "(",
+                    position: node.parameters[0].getStart(),

```

Comment:
You might test there's at least 1 parameter.

Reply:
checked, thanks